<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="openFolderButton">
      <property name="text">
       <string>Open Folder</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="folderPathLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="countButton">
      <property name="text">
       <string>Count Files and Folders</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="folderCountLabel">
      <property name="text">
       <string>Folder Count:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="fileCountLabel">
      <property name="text">
       <string>File Count:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
